pkgs/tools/misc/colord/default.nix

```
@@ -18,14 +18,15 @@
 , ninja
 , vala
 , libgudev
-, wrapGAppsHook
+, wrapGAppsNoGuiHook
 , shared-mime-info
 , sane-backends
 , docbook_xsl
 , docbook_xsl_ns
 , docbook_xml_dtd_412
 , gtk-doc
 , libxslt
+, enableDaemon ? !stdenv.hostPlatform.isMusl || stdenv.hostPlatform.isStatic
```

Does it work on static, or should it be

```suggestion
, enableDaemon ? !(stdenv.hostPlatform.isMusl || stdenv.hostPlatform.isStatic)
```

---

It works on static — in fact the problem is that rustc assumes that "musl" always means static, and that's why it doesn't work with dynamic musl.